yaksh/views.py

```diff
@@ -580,7 +595,7 @@ def add_course(request):
 def enroll_request(request, course_id):
     user = request.user
     ci = RequestContext(request)
-    course = get_object_or_404(Course, pk=course_id)
+    course = get_object_or_404(Course, pk=course_id, is_trial=False)
```

Comment:
This is repeated all over, can there be a convenient function to do this nicely?  The need to set `is_trial=False` everywhere seems a bit disturbing.